plugins/generator-1.14.4/forge-1.14.4/templates/biome.java.ftl

```diff
@@ -158,9 +158,11 @@ package ${package}.world.biome;
 		@OnlyIn(Dist.CLIENT) @Override public int getGrassColor(BlockPos pos) {
 			return ${data.grassColor.getRGB()};

```

Comment:
As you constructed JColor with allowNullColor set to true, wrap this in if statement similar to how other color parameters do already